config/config.php

```diff
@@ -30,6 +30,10 @@
             'links' => [
                 'service' => '/post.php?post=%d&action=edit',
                 'client' => '/edit.php?post_type=download&page=edd-customers&view=overview&id=%d'
+            ],
+            'currency' => [
```

Comment:
I could not understand why `currency` needs to be under a `config` key. I don't know why configuration under `config` is different from any other configuration; why it is grouped under that namespace. In the future, change config structure to be more natural; see [card](https://github.com/orgs/RebelCode/projects/2#card-11125285).